pages/en/developer/assemblyscript-api.mdx

```diff
@@ -357,6 +358,7 @@ class Event {
   block: Block
   transaction: Transaction
   parameters: Array<EventParam>
+  receipt: TransactionReceipt | null
```

Comment:
@tilacog I think we need to add the schema for the TransactionReceipt object here so folks know what they can use? I don't think it is documented yet